engineering/project-template/DCO

```diff
@@ -0,0 +1,36 @@
+Developer Certificate of Origin
```

Comment:
Is the standard for DCO file not carrying the `.md` extension? Asking this because in some cases CONTRIBUTING, CODE_OF_CONDUCT, LICENSE have `.md` extension, others they don't…

Reply:
We're omitting `.md` in `LICENSE ` files (what also causes problems in GitBook documentation). GitHub accepts [both options](https://help.github.com/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository), so since it would also fix the problem with links to License in GitBook, I'd choose the `.md` version

I found no info about `DCO` but it worths to follow GitHub expectations because doing so it will be shown in PRs (for example)

Reply:
Let's move the DCO thing to a separate PR.